packages/core/stories/card/card.doc.stories.mdx

```diff
@@ -26,38 +26,62 @@ This default example is intended to show the way the Card component’s content
   <Story id="core-card--default" />
 </Canvas>
 
-## Interactable
+## Interactable Card
 
-Cards can have interactable styling by setting `interactable={true}`. These cards could be wrapped in Link, for example, for interactivity. We discourage the use of any focusable elements within interactable Cards.
+**Deprecated:** In previous versions, Cards could have interactable styling by setting `interactable={true}`. These cards could be wrapped in Link, for example, for interactivity. We discourage the use of any focusable elements within interactable Cards.
```

Comment:
The **Deprecated** and **Updated** sections in this document seem weird to me and feel like they lack context.
Is it even worth highlighting these as deprecated in the docs? We should improve the changeset by adding rich diffs of how a user would migrate from using the deprecated Card props to the new InteractableCard